tabletop/automated_sms.py

```diff
@@ -0,0 +1,44 @@
+from tabletop import *
```

Comment:
The code in this file isn't generic enough to work off of any kind of general ``AutomatedSMS`` class, but it probably serves as a solid foundation for how we can do this kind of thing in the future.  It's vaguely similar to our ``AutomatedEmail`` implementation.